knox-agent/pom.xml

```diff
@@ -61,6 +61,10 @@
                     <groupId>org.apache.zookeeper</groupId>
                     <artifactId>zookeeper</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>javax.ws.rs</groupId>
+                    <artifactId>jsr311-api</artifactId>
+                </exclusion>
```

Comment:
To avoid conflicts in `RangerAdminJersey2RestClient.java`, exclude it.

```
[INFO] +- org.glassfish.jersey.core:jersey-client:jar:2.35:compile
[INFO] |  +- jakarta.ws.rs:jakarta.ws.rs-api:jar:2.1.6:compile
[INFO] |  +- org.glassfish.jersey.core:jersey-common:jar:2.35:compile
[INFO] |  |  +- jakarta.annotation:jakarta.annotation-api:jar:1.3.5:compile
[INFO] |  |  \- org.glassfish.hk2:osgi-resource-locator:jar:1.0.3:compile
[INFO] |  \- org.glassfish.hk2.external:jakarta.inject:jar:2.6.1:compile
```

`javax.ws.rs.core.Response` on `jakarta.ws.rs-api-2.1.6.jar` has `readEntity()`.

`javax.ws.rs.core.Response` on `jsr311-api-1.1.1.jar` doesn't has `readEntity()`.